### PR TITLE
Adds event tracking when user opens accordions

### PR DESCRIPTION
**Pull request by tonytlwu**
Deals with the following [reported issue](https://docs.google.com/spreadsheets/d/1ITLCZm9fg_A-LIYQDbJsNyTWqPFyF0_25F6-cIR9Z84/edit):

> - accordions - expand/contract not tracked

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -173,7 +173,14 @@
   $(document)
     .on('show.bs.collapse', Collapsible.SELECTORS.collapse, function(){
       // Immediately when the expand action is fired
-      Collapsible.prototype.toggleChevron($(this).attr('id'), true);
+      var id = $(this).attr('id');
+      var label = $('[data-toggle="collapse"][data-target="#' + id + '"]').text().trim();
```
Comment (squallstar) [inline js/build.js:177]: any reason why this would fail? just checking :)

File: js/build.js [hunk 2]
```diff
@@ -173,7 +173,14 @@
...
+      Fliplet.Analytics.trackEvent({
```
Comment (squallstar) [inline js/build.js]: if possible I would always track events after any other action, to make sure the UI feedback happens before

File: js/build.js (hunk 1 above)
Comment (tonytlwu) [reply js/build.js:177]: @squallstar Yup. I've checked.

- There's always going to be an ID.
- There's a possiblity of `$('[data-toggle="collapse"][data-target="#' + id + '"]')` returning nothing. If so, `$('[data-toggle="collapse"][data-target="#' + id + '"]').text().trim()` will still work and label will just be an empty string. Not a problem to the subsequent code.

File: js/build.js (hunk 2 above)
Comment (tonytlwu) [reply js/build.js]: @squallstar Updated 👌

**Review by squallstar — APPROVED** [review]

**Pull request merged**